scripts/generator/templates/template-vaadin-bom.xml

```diff
@@ -15,7 +15,7 @@
     <properties>
 {{javadeps}}
         <slf4j.version>2.0.4</slf4j.version>
```

Comment:
Off Topic: this looks quite old :) 2.0.11 is currently in use in flow / -components 

Additionally, since 2.0.9 you can also use the slf4j-bom instead of managing all dependencies yourself. (Even tho I would ask why Vaadin even tries to enforce a slf4j version to their customer projects :P)

```xml
      <!-- https://central.sonatype.com/artifact/org.slf4j/slf4j-bom -->
      <dependency>
        <groupId>org.slf4j</groupId>
        <artifactId>slf4j-bom</artifactId>
        <version>${slf4j.version}</version>
        <scope>import</scope>
        <type>pom</type>
      </dependency>
```

Reply:
I found exactly the same question from https://github.com/vaadin/platform/issues/49 :-P

Reply:
I will address this version in a different PR, in case different slf4j versions are used in different branches.

Reply:
here is the other one. https://github.com/vaadin/platform/pull/4902